Traceback (most recent call last):
  File "/home/kcb/school/capstone_repo/gymnasium/main.py", line 57, in <module>
    agent = Agent(env)
            ^^^^^^^^^^
  File "/home/kcb/school/capstone_repo/gymnasium/agent.py", line 36, in __init__
    self.actor.compile(optimizer=opt.Adam(learning_rate=self.actor_lr))
  File "/home/kcb/capstone/lib/python3.12/site-packages/keras/src/utils/traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/kcb/capstone/lib/python3.12/site-packages/keras/src/layers/layer.py", line 682, in dtype_policy
    return self._dtype_policy
           ^^^^^^^^^^^^^^^^^^
AttributeError: 'Actor' object has no attribute '_dtype_policy'. Did you mean: 'dtype_policy'?